### DISCUSSION TITLE
How to facilitate a TensorFlow Lite Model

### DISCUSSION THREAD

**Discussion by MisterK91** [deepjavalibrary/djl#1660] [Q&A] [answered]
Hello folks, 
actually I noticed your work and really love it as I come from a Java oriented world. 
Recently I tried to use a pre-trained TensorFlow Lite Model, but unfortunatelly I did not found any examples or prober documentation of loading it. 

The model itself is a common predict/classifier model and uses a float array filled with a sound signal. 

It seems to be a less common use case but would be really thankful if I someone can assist me with the set-up.

Cheers, Kevin

**Comment by frankfliu** [comment, marked as answer]
@kliebhart 

Here is Image classification example for TFLite: https://github.com/deepjavalibrary/djl/blob/master/engines/tflite/tflite-engine/src/test/java/ai/djl/tflite/engine/TfLiteTest.java#L41-L52

You can see it's almost exactly the same as using other engine.

In your case, you need create your own `Translator<byte[], Classifications>` class and hand pre/post processing to match your model.
